packages/user-group-service/src/groups/typedef.graphql

```diff
@@ -2,11 +2,23 @@ type Query {
   """
   Returns a list of all Groups (default limit=25)
   """
-  listGroups(limit: Int): [Group]
+  listGroups(limit: Int): [Group] @deprecated(reason: "This query is no longer maintained")
   """
   Returns a list of Groups for the given selector query (default limit=25)
   """
-  getGroupsBy(selector: GetGroupInput!, limit: Int): [Group]
+  getGroupsBy(selector: GetGroupInput!, limit: Int): [Group] @deprecated(reason: "This query is no longer maintained")
+  """
+  Returns a group
+  """
+  group( cn: String ): Group
+  """
+  Returns a list of Groups for the given selector query (default limit=25)
+  """
+  groups(selector: GetGroupInput!, limit: Int): [Group]
+  """
+  Returns a list of Group Names for the given user
+  """
+  groupsByUser( uid: String ): [String]
```

Comment:
Not needed, as this can be obtained from the user query.

Convert this into a subquery in the `RoverUser` type.